tools/autograd/derivatives.yaml

```diff
@@ -98,11 +98,9 @@
 - name: alias(Tensor self)
   self: grad
 
-- name: all  # fallthrough
-
-- name: any  # fallthrough
-
-- name: arange  # fallthrough
+# all
```

Comment:
What determines whether I should put a comment with my function here or not?  Certainly we don't list all of them (e.g. allclose).

Reply:
Currently, it's the TH operations that don't (and shouldn't) have derivatives specified. I'm fine with deleting them now.

Reply:
Sure, let's delete them.